modules/upgrade/pages/upgrade-script-example.adoc

```diff
@@ -179,34 +230,25 @@ Use this script as the basis for an upgrade from SLE{nbsp}15 to SLE{nbsp}15.1.
         </source>
       </script>
     </chroot-scripts>
+    <init-scripts config:type="list">
+      $SNIPPET('spacewalk/minion_script')
+<!-- when client is traditional registered, please use this snippet
+      $SNIPPET('spacewalk/sles_register_script')
+-->
+    </init-scripts>
   </scripts>
 </profile>
 ----
 
 
 
-== Upgrading an SAP Client from SLE{nbsp}15 to SLE{nbsp}15.1
+== Upgrading an SAP Client from SLE{nbsp}12{nbsp}SPn to SLE{nbsp}15{nbsp}SP1
```

Comment:
I'm not sure it's clear what SPn would mean. Can we just say "SLE{nbsp}12" without specifying an SP?

Reply:
I think you and @keichwa should come to a conclusion how many "version" we should name here.

The "source" does not matter. Only the target version is important as the examples migrate to a specific version.
So if we say just SLE12 or SLE12 SPn or nothing is something you need to decide.

Reply:
I would prefer to remove the versioning as much as possible from this entire guide, but that's for a different PR. For now, let's just leave it as SLE 12.